lib/developer_portal/app/controllers/developer_portal/base_controller.rb

```diff
@@ -21,7 +21,8 @@ def require_credit_card_on_signup?
   end
 
   def finish_signup_for_paid_plan
-    return if unacceptable_payment_gateway?(site_account) || !require_credit_card_on_signup?
```

Comment:
where was this method defined? I think deginition needs to be removed.

Reply:
It has been removed: https://github.com/3scale/porta/pull/3171/files/523a027a8c0bd2acabd598745c63785e2b298e1e#diff-9c5e547a8ac256ed8540b94e2432530835017daa884433c71b40ad1e0d4fdcabL21